### DISCUSSION TITLE
0.9.4+msmtpfix

### DISCUSSION THREAD

**Discussion by ckulka** [ckulka/baikal-docker#150] [Announcements]
## What's Changed

* Fix msmtprc file permissions by @thomasjanker in https://github.com/ckulka/baikal-docker/pull/142
* Add msmtp test automation and updated dependencies by @ckulka in https://github.com/ckulka/baikal-docker/pull/143
* Retry mail test automation to address flakiness by @ckulka in https://github.com/ckulka/baikal-docker/pull/146
* Check lint result during test workflow by @ckulka in https://github.com/ckulka/baikal-docker/pull/145
* Create dependabot.yml by @ckulka in https://github.com/ckulka/baikal-docker/pull/147
* Re-release 0.9.4 to include msmtp fix by @ckulka in https://github.com/ckulka/baikal-docker/pull/148

## New Contributors

* @thomasjanker made their first contribution in https://github.com/ckulka/baikal-docker/pull/142

**Full Changelog**: https://github.com/ckulka/baikal-docker/compare/0.9.4...0.9.4+msmtpfix

<hr /><em>This discussion was created from the release <a href='https://github.com/ckulka/baikal-docker/releases/tag/0.9.4%2Bmsmtpfix'>0.9.4+msmtpfix</a>.</em>

**Comment by MrPinrel** [comment]
I would like to try this release but it is not showing as a downloadable option from the docker registry. What is the easiest way to get it installed?
